hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -208,6 +247,31 @@ public static GenericRecord generateGenericRecord(String rowKey, String riderNam
     return rec;
   }
 
+  /*
+  Generate random record using TRIP_UBER_EXAMPLE_SCHEMA
+   */
+  public GenericRecord generateRecordForUberSchema(String rowKey, String riderName, String driverName, double timestamp) {
+    GenericRecord rec = new GenericData.Record(AVRO_TRIP_SCHEMA);
+    rec.put("_row_key", rowKey);
+    rec.put("timestamp", timestamp);
+    rec.put("rider", riderName);
+    rec.put("driver", driverName);
+    rec.put("fare", RAND.nextDouble() * 100);
+    rec.put("_hoodie_is_deleted", false);
+    return rec;
+  }
+
+  public GenericRecord generateRecordForFgSchema(String rowKey, String riderName, String driverName, double timestamp) {
```

Comment:
Same rename here. Please check for other method names as well.

Reply:
Done.